src/FormBuilder/resources/js/form-builder/src/components/canvas/FormSettingsContainer/components/Menu.tsx

```diff
@@ -0,0 +1,63 @@
+import {__} from '@wordpress/i18n';
+import {chevronLeft, chevronRight} from '@wordpress/icons';
+import {Icon} from '@wordpress/components';
+import {useFormSettingsContext} from '@givewp/form-builder/components/canvas/FormSettingsContainer';
+import {
+    navigateBackInMenu,
+    updateMenuState,
+} from '@givewp/form-builder/components/canvas/FormSettingsContainer/formSettingsReducer';
+import classnames from 'classnames';
+
+function MenuItem({item}) {
```

Comment:
Can we add this `item` as a typescript type so the rest of the logic has context?

Reply:
Resolved: 31e72b285a141659740f4ae14cb86198d86811e7